tools/topology/topology1/CMakeLists.txt

```diff
@@ -305,8 +305,11 @@ set(TPLGS
 	"sof-rn-rt5682-max98360\;sof-rn-rt5682-max98360"
 	"sof-acp-rmb\;sof-acp-rmb"
 	"sof-acp-rmb-dmic4ch\;sof-acp-rmb-dmic4ch"
+	"sof-acp-rmb-tdm8ch-dmic4ch\;sof-acp-rmb-tdm8ch-dmic4ch"
+	"sof-acp-rmb-tdm8ch-dmic2ch-nuvoton\;sof-acp-rmb-tdm8ch-dmic2ch-nuvoton"
+	"sof-acp-rmb-tdm8ch-dmic4ch-nuvoton\;sof-acp-rmb-tdm8ch-dmic4ch-nuvoton"
```

Comment:
nuvoton sounds awfully vague and broad. This is a topology name that will be used by the kernel driver, shouldn't this be more specific of a nuvoton chip or family?

Reply:
added chip specific info